src/safeds/ml/nn/layers/_dropout_layer.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from safeds._utils import _structural_hash
+from safeds._validation import _check_bounds, _OpenBound
+from safeds.ml.nn.typing import ModelImageSize
+
+from ._layer import Layer
+
+if TYPE_CHECKING:
+    from torch import nn
+
+
+class DropoutLayer(Layer):
+    """
+    Create a dropout Layer.
+
+    Parameters
+    ----------
+    probability:
+        The probability of which the input neuron becomes an output neuron
```

Comment:
It's the opposite of this: This is the probability that an input element is replaced by 0.